tasks/9_find-numbers/find-numbers.js

```diff
@@ -0,0 +1,10 @@
+function getNumbers(str) {
+  const regex = /\d+/g;
+  let digitsArr = str.match(regex);
+  for (let index = 0; index < digitsArr.length; index++) {
+    digitsArr[index] = +digitsArr[index];
```

Comment:
Please use `Number(val)` to cast a value to a number, it clearly shows your intent